osquery/logger/logger.cpp

```diff
@@ -326,8 +317,6 @@ void initLogger(const std::string& name) {
     return;
   }
 
-  // Stop the buffering sink and store the intermediate logs.
-  BufferedLogSink::get().disable();
```

Comment:
No need to call disable. active_ is false anyway.

disable is called to make sure that, no log is sent from BufferedLogSink to the plugins before they all are initialized.(and logs will be buffered) 

enabled_ is checked in BufferedLogSink::send